apps/certs/tests/test_models.py

```
@@ -37,6 +37,37 @@ def fake_request_certbot_certificate(**kwargs):
+    now = timezone.now()
+    monkeypatch.setattr(timezone, "now", lambda: now)
+
+    output = "\n".join(
+        [
+            "Successfully received certificate.",
+            "Certificate is saved at: /etc/letsencrypt/live/example.com-0001/fullchain.pem",
+            "Key is saved at: /etc/letsencrypt/live/example.com-0001/privkey.pem",
+        ]
+    )
+
+    monkeypatch.setattr(services, "request_certbot_certificate", lambda **kwargs: output)
```

Ruff (0.15.1)

[warning] Unused lambda argument: `kwargs`

(ARG005)